### PR TITLE
Implement support for boss level `No Need for Introductions`

### PR DESCRIPTION
**Pull request by jbzdarkid**

### CODE REVIEW

**Comment by jbzdarkid** [comment]
Reminder from 12345ieee -- please don't forget to squash-merge this PR, to keep the history clean.

**Comment by jbzdarkid** [comment]
[test_spam.txt](https://github.com/spacechem-community-developers/SChem/files/11730157/test_spam.txt)
Attaching the exported solutions from 12345ieee -- I went through about 25% of it, found one bug (c566d67) but it's been green since then, and I'm not sure how much more time I want to invest here.

File: tests/test_solution.py [hunk 1]
```diff
@@ -225,6 +225,16 @@ def test_evaluate_duplicate_levels_success(self):
 
         assert json == expected_json, f"Expected:\n{expected_json}\nbut got\n{json}"
 
+    def test_death_solutions(self):
```
Comment (zig1000) [inline tests/test_solution.py:228]: Duplicated above from the other PR; remove

Comment (jbzdarkid) [reply tests/test_solution.py]: Fixed

**Pull request merged**